pkg/controller/migcluster/storageclass.go

```diff
@@ -0,0 +1,74 @@
+package migcluster
+
+import (
+	"strconv"
+
+	migapi "github.com/fusor/mig-controller/pkg/apis/migration/v1alpha1"
+	kapi "k8s.io/api/core/v1"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Sets the list of storage classes from the cluster
+func (r *ReconcileMigCluster) setStorageClasses(cluster *migapi.MigCluster) error {
+	var client k8sclient.Client
+	client, err := cluster.GetClient(r)
+	if err != nil {
+		log.Trace(err)
+		return err
+	}
+	clusterStorageClasses, err := cluster.GetStorageClasses(client)
+	if err != nil {
+		log.Trace(err)
+		return err
+	}
+	var storageClasses []migapi.StorageClass
+	for _, clusterStorageClass := range clusterStorageClasses {
+		storageClass := migapi.StorageClass{
+			Name:        clusterStorageClass.Name,
+			Provisioner: clusterStorageClass.Provisioner,
+			AccessModes: r.accessModesForProvisioner(clusterStorageClass.Provisioner),
+		}
+		if clusterStorageClass.Annotations != nil {
+			storageClass.Default, _ = strconv.ParseBool(clusterStorageClass.Annotations["storageclass.kubernetes.io/is-default-class"])
```

Comment:
@dymurray I can return the error here. It just felt like failing reconcile over it was overkill, since if we can't parse the "is default?" boolean, it was safe to assume that the storageclass *isn't* the default. Is there a downside to this, though?

Reply:
Fair, I don't see a downside at the moment. I think it's fine to eat it then